tycho/tests/routes/test_event.py

```diff
@@ -293,7 +293,7 @@
 async def test_put_invalid_event_raises_exception(cli):
     resp = await cli.put('/api/v1/event/',
                          headers={"content-type": "application/json"},
-                         data=json.dumps({"event": {"bad_key": "hello"}})
+                         data=json.dumps({"event": {"start_time": "hello"}})
```

Comment:
Shouldn't we also be rejecting events with malformed keys?

Reply:
yeah that's tricky. I think if we do that, we can't rollback changes that add new keys without breaking consumers who are publishing data with new keys.

This means people will be broken if a key is used in an unintended way, but I think that's a better of the two edge cases.